### PR TITLE
fix(app-selector): Remove direct use of strings 

### PR DESCRIPTION
**Pull request by TheAabedKhan**

### CODE REVIEW

File: lib/ui/widgets/appSelectorView/not_installed_app_item.dart [hunk 1]
```diff
@@ -54,12 +55,28 @@ class _NotInstalledAppItem extends State<NotInstalledAppItem> {
...
                     children: [
-                      Text(
-                        'Suggested: ${widget.suggestedVersion.isEmpty ? 'All versions' : 'v${widget.suggestedVersion}'}',
+                      I18nText(
+                        FlutterI18n.translate(
+                          context,
+                          'suggested',
+                          translationParams: {
+                            'version': widget.suggestedVersion.isEmpty
+                                ? 'All versions'
```
Comment (Ushie) [inline lib/ui/widgets/appSelectorView/not_installed_app_item.dart:76]: Did you happen to miss using the key here instead?

**Review by Ushie — CHANGES_REQUESTED** [review]

File: lib/ui/widgets/appSelectorView/not_installed_app_item.dart (hunk 1 above)
Comment (TheAabedKhan) [reply lib/ui/widgets/appSelectorView/not_installed_app_item.dart:76]: Which key is missing?

Comment (Ushie) [reply lib/ui/widgets/appSelectorView/not_installed_app_item.dart:76]: You could create an i18n key for the string "All versions" here

Comment (TheAabedKhan) [reply lib/ui/widgets/appSelectorView/not_installed_app_item.dart]: Can you check the new commits?

File: assets/i18n/en_US.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "enabledLabel": "Enabled",
     "disabledLabel": "Disabled",
     "installed":"Installed: {version}",
-    "suggested":"Suggested: {version}",
+    "suggested":"Suggested: ",
```
Comment (Ushie) [inline assets/i18n/en_US.json:8]: Did you remove `{version}` by mistake? or intentionally

**Review by Ushie — CHANGES_REQUESTED** [review]

File: assets/i18n/en_US.json (hunk 2 above)
Comment (TheAabedKhan) [reply assets/i18n/en_US.json:8]: Intentionally, as it is no longer required anywhere.

**Review by Ushie — CHANGES_REQUESTED** [review]

File: assets/i18n/en_US.json [hunk 3]
```diff
@@ -58,9 +58,11 @@
         "widgetTitleSelected": "Selected application",
         "widgetSubtitle": "No application selected",
         "noAppsLabel": "No applications found",
+        "notInstalled":"App not installed",
         "currentVersion": "Current",
         "suggestedVersion": "Suggested",
-        "anyVersion": "any"
+        "anyVersion": "any",
+        "allVersions": "All versions"
```
[the remark this answers is not in the document]
Comment (TheAabedKhan) [reply assets/i18n/en_US.json:65]: The `anyVersion` is used in the Patcher page after you select any application for patching. I also thought if I could replace it with `allVersions`.  Let me know what you think.

**Pull request merged**